modules/ztp-creating-ztp-custom-resources-for-multiple-managed-clusters.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-creating-ztp-custom-resources-for-multiple-managed-clusters_{context}"]
+= Creating ZTP custom resources for multiple managed clusters
+
+If you are installing multiple managed clusters, zero touch provisioning (ZTP) uses ArgoCD and SiteConfig to manage the processes that create the custom resources and generate and apply the policies for multiple clusters, in batches of no more than 100, using the GitOps approach.
+
+Installing and deploying the clusters is a two stage process, as shown here:
+
```

Comment:
I would suggest adding an image title before the image

Reply:
And perhaps also add the procedure module formatting

Reply:
Not needed.